website/package.json

```diff
@@ -2,6 +2,7 @@
   "scripts": {
     "examples": "docusaurus-examples",
     "start": "docusaurus start",
+    "serve": "docusaurus start",
```

Comment:
just wondering why we need to have both 'start' and 'serve'

Reply:
because after `yarn build` is run, it suggests running docusaurus with `serve`, it took me some research to figure out that `start` is the keyword to use. While adding an alias might seem superfluous, I think it would have saved me a few minutes.